samples/nrf9160/modem_shell/src/gnss/gnss.c

```diff
@@ -853,6 +860,10 @@ static void inject_pgps_data_work_fn(struct k_work *work)
 static void pgps_event_handler(struct nrf_cloud_pgps_event *event)
 {
 	switch (event->type) {
+	case PGPS_EVT_READY:
+		if (event->prediction == NULL) {
+			break;
+		} /* else fall through */
```

Comment:
nit: I gather that you don't need to check `event->prediction == NULL` if the event type is `PGPS_EVT_AVAILABLE`, but it may make the code slightly more readable to move this check into `PGPS_EVT_AVAILABLE` anyways, rather than rely on the fall-through